ovos_core/skill_manager.py

```
@@ -16,12 +16,14 @@
 import os
 from os.path import basename
 from threading import Thread, Event, Lock
-from time import sleep, monotonic
 
-from ovos_backend_client.pairing import is_paired
 from ovos_bus_client.apis.enclosure import EnclosureAPI
 from ovos_bus_client.client import MessageBusClient
 from ovos_bus_client.message import Message
+from ovos_workshop.skill_launcher import SKILL_MAIN_MODULE
+from ovos_workshop.skill_launcher import SkillLoader, PluginSkillLoader
+from time import sleep, monotonic
```

Ruff

`time.sleep` imported but unused

Remove unused import: `time.sleep`

(F401)